### ISSUE TITLE
App broken in iOS 18.1 Developer Beta

### ISSUE THREAD

**Issue by chentex** [bitwarden/ios#891]
### Bitwarden Beta

- [ ] I'm using the new native Bitwarden Beta app and I'm aware that legacy .NET app bugs should be reported in [bitwarden/mobile](https://github.com/bitwarden/mobile)

### Steps To Reproduce

1. Go to 'Open the App
2. Log in
3. Error message appears as "An error has occurred"
4. Tap on 'Ok' to scknowledge

### Expected Result

Should be able to see all the passwords.

### Actual Result

Nothing loads.

### Screenshots or Videos

![photo_2024-09-04_18-57-35](https://github.com/user-attachments/assets/1139804b-c04d-4e3c-b411-77d3da0c4ff2)
![photo_2024-09-04_18-57-36](https://github.com/user-attachments/assets/c78c8a6e-07fd-460c-8f16-65145419276e)

### Build Version

2024.8.1

### Environment Details

- Device: iPhone 15 Pro Max
- OS: 18.1 Developer Beta

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by aksdb** [comment]
I have the same issue on iOS 17.6.1. The old iOS app (the .NET one) worked fine.

**Comment by Neonwarden** [comment]
Hi there,

This has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by aksdb** [comment]
Well... is there a way to get more details? The error message isn't exactly helpful.

**Comment by chentex** [comment]
I was able to ge tthe same error, trying to do a manual Sync, so points, and I am guessing when it logs in it also tries to sync first and the error comes from there.

**Comment by ClarkAllen1556** [comment]
I experienced this issue as well (on normal release branch of iOS 17.6.1), after updating the Bitwarden app to 2024.8.1. Logging out/in of my Bitwarden account seemed to fix the issue.

**Comment by janrudolph** [comment]
Edit: I am also using iOS 17.6.1 and I am trying to connect to a self-hosted server

**Comment by Grizaster** [comment]
like @janrudolph , iOS 17.6.1 (iPhone 12) and trying to connect to a self-hosted server. Get the error after entering Email-Address.

**Comment by aksdb** [comment]
To add to my previous question:  is there a more detailed error message in a log that can be found somewhere on the iPhone? If not: would it be possible to add more details to the error messages in a future release to help narrow down the root cause?

**Comment by zhheo** [comment]
I'm having this problem too, I'm on a self-hosted server, iOS 18.0beta

**Comment by janrudolph** [comment]
I found the issue: my self-hosted server was not up to date. After upgrading it seems to work, have not tested every client yet.

**Comment by chentex** [comment]
I tried what @ClarkAllen1556 said, removed the accounts from the app, and added them again and now it is working.

I'll leave it up to the team if they close the issue or not.

**Comment by devnull-hub-lab** [comment]
Hello,

I attached some debug logs here: https://github.com/bitwarden/ios/issues/833#issuecomment-2338904219

**Comment by Najki** [comment]
> I found the issue: my self-hosted server was not up to date. After upgrading it seems to work, have not tested every client yet.

I've had the exact same issues as reported in this thread and @janrudolph's advice to ugrade my Vaultwarden did the trick.

**Comment by devnull-hub-lab** [comment]
Yep, indeed @Najki , thanks for the tip.
I upgrade my bitwarden server, and the new Bitwarden App is working fine now!

@Neonwarden, It seems it was solved. I think the issue can be closed.

**Comment by Najki** [comment]
I'm not so sure if closing closing this issue is a good idea. The old version of Bitwarden iOS app did work. The new does not.

**Comment by BrandonShega** [comment]
I'm having this issue on latest iOS 18 beta even with the latest version of vaultwarden server.

**Comment by nicolasluckie** [comment]
I recently updated the Bitwarden iOS app to 2024.8.1 (1052) and started seeing this issue.
I fixed it by pulling the latest vaultwarden image and upgrading my server from 2024.1 to 2024.6.2.

**Comment by jtxnn678** [comment]
Selfhosted on: Proxmox Debian 12 VM
Vaultwarden Docker image: Master branch, latest (as of Sep. 21, 2024)
Bitwarden Client: Version 2024.9.1
iPhone 15 Pro: iOS 18.0

It was running fine for more than a year, suddently stopped working after upgrade iPhone to iOS 18. Bitwarden Browser plugin and Bitwarden clients (Linux/Windows) are all working perfectly.

**Can't login to Bitwarden Client on iPhone (iOS 18.0 update)**
- Bitwarden was logged in but not display any password, refresh or re-sync got message "An error has occurred."
- Logged out, then tried to re-login with correct credentials, also got "An error has occurred."
- Updated Debian VM, pulled off docker image of Vaultwarden server to lastest, rebooted docker VM. - still says "An error has occurred."
- Deleted Bitwarden client from iPhone, turned off phone and restarted. - still says "An error has occurred."

Tried above recommended fix but didn't work.

Anybody has insight?

**Comment by Kimiblock** [comment]
I'm using iOS 18 release with the latest Bitwarden, also got the error message

**Comment by lihao20050525** [comment]
True, Update to the latest version of Vaultwarden Server, Problem solved.

**Comment by nkasco** [comment]
I am also having this issue. iOS 18.1 Public Beta
Noticed my vault wasn't syncing since 9/14 (probably when I installed either iOS 18 GA or iOS 18.1 Public Beta), tried to sync and you get "An error has occurred"

Logout/Login, unable to login again
Deleted app, updated self-hosted instance, Chrome extension on Windows working fine, iOS just seems to not be playing nice.

I couldn't find anything backing up to iCloud, so rule that out.

**Comment by nkasco** [comment]
I hooked my phone up to XCode and pulled the error from the console. How do I proceed?

error 18:54:36.527312-0400 Bitwarden -[RTIInputSystemClient remoteTextInputSessionWithID:performInputOperation:] perform input operation requires a valid sessionID. inputModality = Keyboard, inputOperation = , customInfoType = UIEmojiSearchOperations

**Comment by SergeantConfused** [comment]
Hello everyone,

Is anyone still encountering this with recent iOS and [Bitwarden](https://bitwarden.com/help/releasenotes/) versions?


Thank you,

**Comment by nkasco** [comment]
> Hello everyone,
> 
> Is anyone still encountering this with recent iOS and [Bitwarden](https://bitwarden.com/help/releasenotes/) versions?
> 
> 
> Thank you,

Sure am, and nobody seems to want to help. Just quick "couldn't reproduce in lab" responses. 

Pretty sad that we can get some actual support.

**Comment by BrandonShega** [comment]
> Hello everyone,
> 
> Is anyone still encountering this with recent iOS and [Bitwarden](https://bitwarden.com/help/releasenotes/) versions?
> 
> Thank you,

I am not experiencing this anymore.  I'm currently on iOS 18.3 beta and everything seems to be working fine.

**Comment by serafearz** [comment]
on iOS 26 here, i get the same error on phone, how to fix?

**Comment by fedemkr** [comment]
Hi there, thanks for reporting this. Could you share app version and whether you are using the "Bitwarden" or "Bitwarden Beta" app? You can get this quickly by going to _Settings -> About -> Version (tap on the copy icon on the right)_.
Moreover, are you Self-hosting or using non-Bitwarden cloud servers (like Vaultwarden)?

Additionally, If possible could you use **Flight Recorder** (help on usage [here](https://bitwarden.com/help/flight-recorder/)) and send the logs to our customer support so we can further investigate this with more contextual information?

Is the issue resolved if you log out and log back in?

**Comment by serafearz** [comment]
It was solved when i used a lets encrypt certificate without browser warnings.
